SunEngine/ServerStartup.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using NLog;
+
+
+namespace SunEngine
+{
+    public class ServerStartup
+    {
+        public void RunServer(StartupConfiguration startupConfiguration)
+        {
+            var webHost = CreateWebHostBuilder(startupConfiguration).Build();
+                
+            IHostingEnvironment env = (IHostingEnvironment) webHost.Services.GetService(typeof(IHostingEnvironment));
+            IConfiguration conf = (IConfiguration) webHost.Services.GetService(typeof(IConfiguration));
+
+            // TODO refactor this behaviour. Startup commands should not behave differently based on environment. 
+            if (startupConfiguration.StartServer || env.IsDevelopment())
```

Comment:
This condition checks if the "server" argument was passed - if it is, we will start the server.
For dev, Dmitrii proposed to add the possibility to omit this argument and use dotnet run without it to run the server.